modules/sdk-coin-celo/src/lib/testnetTransactionBuilder.ts

```diff
@@ -0,0 +1,33 @@
+import * as ethUtil from 'ethereumjs-util';
+import { Transaction } from '@bitgo/abstract-eth';
+import { BaseCoin as CoinConfig } from '@bitgo/statics';
+import { TransactionBuilder } from './transactionBuilder';
+
+export class TestnetTransactionBuilder extends TransactionBuilder {
+  constructor(_coinConfig: Readonly<CoinConfig>) {
+    super(_coinConfig);
+    this.transaction = new Transaction(_coinConfig, this._common);
+  }
+
+  /**
+   * Get the final v value. Final v is described in EIP-155.
+   *
+   * @protected for internal use when the enableFinalVField flag is true.
+   */
+  protected getFinalV(): string {
+    return ethUtil.addHexPrefix(this._common.chainIdBN().muln(2).addn(35).toString(16));
+  }
+
+  /** @inheritdoc */
+  protected fromImplementation(rawTransaction: string): Transaction {
```

Comment:
This method is almost similar in the celo transaction builder, what is the reason to add this here?